content/en/blog/news/2021-no-docker.md

```diff
@@ -0,0 +1,101 @@
+
+## Why use kubernetes instead of docker?
+
+* why test on a completely different VM and container orchestrator than production? its better to test on a similar environment to where you are really going to deploy your code
+* test your kubernetes yaml / helm chart and associated configuration at the same time as you run your containers helps you catch mistakes earlier
+    * it's not just about running the container image; it's about lots of other things too like networking, configuration, secrets, storage/volumes, cloud infrastructure, service mesh, liveness/readiness/startup probes - so why not test all of those things rather than just the image?
+* some corporate environments don't let you run VMs on your laptop anyway so running docker locally isn't an option
+
+## How to get kubernetes?
+
+First you'll need a kubernetes cluster.
+ 
+I fully agree with James Ward that [developers should not need to run kubernetes](https://twitter.com/_JamesWard/status/1393270529474408450?s=20). Friends don't let friends setup and manage kubernetes clusters by hand :). 
+
+So try ask your infrastructure team for a development cluster or, if you can, use the cloud to set-up a managed kubernetes cluster. All the public clouds have a relatively straightforward way to spin up a fully managed kubernetes cluster for you that will be relatively inexpensive & they are easy to scale down when you don't need them. 
+
+e.g. on [Google Cloud](https://cloud.google.com/kubernetes-engine) it's a couple of clicks and about 5 minutes later you'll have a fully managed kubernetes cluster ready to use. Its easy to enable auto-scaling too. Plus there's a free tier. 
+
+If that's not easy for you to achieve you could try reuse a namespace in your staging cluster? Though we don't recommend developing on a production cluster; its too easy to accidentally mess up production; e.g. by using up too many resources or overwriting a cluster scoped resource like a `CustomResourceDefinition` or `ClusterRole` etc.
+
+If you have zero budget you could try [minikube](https://minikube.sigs.k8s.io/docs/start/) or [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) on top of docker; though its much better to reuse as close to the production setup of kubernetes as you can - there can be large variations in platform, version, setup, network, machine size and so forth.
+
+If you are deploying software on kubernetes then I'd hope you've some managed kubernetes solution; so why not use that and spin up another cluster for your team for development?
+
+If your budget is so stretched that you can't afford 2 kubernetes clusters; one for production and one for development + staging; maybe it's time to look at using just pure serverless / FaaS instead of kubernetes anyway? Even in that case its better to use your serverless / FaaS infrastructure than docker locally for similar reasons                    
```

Comment:
```suggestion
If your budget is so stretched that you can't afford 2 kubernetes clusters; one for production and one for development + staging; maybe it's time to look at using just pure serverless / FaaS instead of kubernetes anyway? Even in that case its better to use your serverless / FaaS infrastructure than docker locally for similar reasons.  
```